test/cypress/integration/simplified-account/service-settings/stripe-details/company-number.cy.js

```diff
@@ -134,6 +134,12 @@ describe('Stripe details settings', () => {
 
           cy.get('.govuk-error-summary').should('not.exist')
 
+          cy.get('input[type="radio"]')
+            .siblings('label')
+            .contains('Yes')
+            .prev('input[type="radio"]')
+            .check()
```

Comment:
I might be misunderstanding something but why are you checking the radio button then clearing it a few lines down on line 144?